routing/pathfind_test.go

```diff
@@ -1203,7 +1203,7 @@ func TestNewRoute(t *testing.T) {
 }
 
 func TestNewRoutePathTooLong(t *testing.T) {
-	t.Skip()
+	t.Parallel()
```

Comment:
This needs an explanation in the commit message, why this can be enabled now.

Reply:
Fixed

Reply:
So enabling this test is unrelated to the other changes in this PR?

Reply:
Right, it doesn't belong in this PR

Reply:
Removed the commit that added this test